### DISCUSSION TITLE
What is the best way to overlay an image?

### DISCUSSION THREAD

**Discussion by oneseventeen** [ValentinH/react-easy-crop#471] [Q&A]
I would like to have a PNG with alpha transparency overlaid on top of the crop tool to allow users to position images (for me, this would be for custom shaped stickers), then I'll just pull the crop data from onCropAreaChange and store the original image and the position/scale to then accurately re-create the image the user saw on their screen to generate the final artwork to be printed.

This would basically be like replacing the grid with a PNG, and I wasn't sure if there is a simple way to do this inside the component, or if I would just do this in CSS outside the component itself.

Thanks!

**Comment by ValentinH** [comment]
You can probably override the crop area style and apply a background-image. For this you can use the classnames option with the `cropAreaClassName` key or override the global classname of the crop area.
